docs/questionSharing.md

```diff
@@ -51,17 +53,22 @@ Just as anyone with access to a question in your course can access any file in `
 
 ## Steps to share a question to a course, using a sharing set
 
-1. On your course admin page, visit the 'sharing' tab
-2. Choose a sharing name for your course
-3. Create a sharing set
-4. Have the instructor or the course you would like to share your question with visit the 'sharing' tab on their course admin page and provide you with their course's sharing token
-5. Use the provided sharing token to add the other instructor's course as a consumer of the sharing set you created
-6. Visit the question settings page for the question you would like to share, and add it to the sharing set
-7. The course you have shared the question with may now use it by referencing it in their assessments
+**Note:** Once a question is added to a sharing set, the question cannot be renamed, deleted, or removed from the sharing set because doing so could break the assessments of people who have used the shared question.
+
+1. On your course admin page, visit the 'sharing' tab.
+2. Choose a sharing name for your course.
+3. Create a sharing set by adding it to your course's `infoCourse.json` file.
+4. Have the instructor or the course you would like to share your question with visit the 'sharing' tab on their course admin page and provide you with their course's sharing token.
+5. Use the provided sharing token to add the other instructor's course as a consumer of the sharing set you created.
+6. Add the sharing set to the `sharingSets` list in `info.json` file of the questions you would like to share.
+7. The course you have shared the question with may now use it by referencing it in their assessments.
 
 ## Steps to share a question publicly
 
-1. On your course admin page, visit the 'sharing' tab
-2. Choose a sharing name for your course
-3. Visit the question settings page for the question you would like to share, click the button to "Share publicly", and complete the confirmation dialog
-4. Anyone with a PrairieLearn account may preview your question, and any PrairieLearn course may now use it by referencing it in their assessments
+**Note:** Once a question is publicly shared, the question cannot be renamed, deleted, or un-publicly shared because doing so could break the assessments of people who have used the shared question. Sharing the source code of a question may be un-done.
+
+1. On your course admin page, visit the 'sharing' tab.
```

Comment:
Extremely a nit, but on pretty much every other page but this one we use double quotes for things like this. Grep for `[a-zA-Z] '[a-zA-Z]` and `[a-zA-Z] "[a-zA-Z]` in `docs/` and compare.